LiteDB/Engine/Disk/Serializer/BufferReader.cs

```diff
@@ -161,13 +163,13 @@ public string ReadString(int count)
             else
             {
                 // rent a buffer to be re-usable
-                var buffer = BufferPool.Rent(count);
```

Comment:
Does using a [BufferOwner](https://gist.github.com/JKamsker/27f65465d401b03e68f3822406227285) make sense here?

Reply:
no, because it will be boxed and will allocate, and here I'm trying to avoid allocations. When we bump the TFM we may use `ref struct` and better APIs